README.md

```diff
@@ -140,6 +140,14 @@ no_proxy: *.example.local,*.other.global,192.168.0.0/16
 no_proxy: long.domain.name.local,example.local
 ```
 
+## Changing default log level ##
```

Comment:
need to add this also to https://github.com/snyk/kubernetes-monitor/blob/staging/snyk-monitor/README.md with concrete helm example (this file is for YAML installation)